CHANGELOG.md

```diff
@@ -6,6 +6,10 @@
 
 - Improved error message to remind users set `{"infer_schema": True}` when reading csv file without specifying its schema.
 
+### Bug Fixes
+
+- Fixed a bug where `Session.create_dataframe` is called with more than 512 rows throws `TypeError` when creating a session with `format` or `pyformat` `paramstyle`.
```

Comment:
just reading the code, why it's the number `512` that triggers the error?

Reply:
512 is set as `analyzer.ARRAY_BIND_THRESHOLD`